tests/orquestra/opt/optimizers/simple_gradient_descent_test.py

```diff
@@ -62,14 +62,13 @@ def test_fails_to_initialize_when_number_of_iterations_is_negative(self):
         with pytest.raises(AssertionError):
             SimpleGradientDescent(0.1, -1)
 
-    def test_fails_to_minimize_when_cost_function_does_not_have_gradient_method(
+    def test_succeeds_to_minimize_when_cost_function_does_not_have_gradient_method(
```

Comment:
```suggestion
    def test_minimize_succeeds_when_cost_function_does_not_have_gradient_method(
```
Just a small chang here for clarity.